robot/Cumulus/tests/browser/gift_entry/review_donations_create_opp_sge.robot

```diff
@@ -0,0 +1,72 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
```

Comment:
Are you using this page object in your test ?

Reply:
yes